src/Controls/StreetControls.js

```diff
@@ -77,8 +113,74 @@ class StreetControls extends FirstPersonControls {
         this.previousPosition = newPreviousPosition;
     }
 
+    pick(event, pickGround = () => {}, pickObject = () => {}, target = new THREE.Vector3()) {
+        // get real distance to ground, with a specific method to pick on the elevation layer
+        this.view.getPickingPositionFromDepth(this.view.eventToViewCoords(event), target);
```

Comment:
It would be necessary to open an issue to add the distance calculation in `Picking.pickTilesAt`.

Reply:
ok #1037